language_model/tensorflow/bert/run_pretraining.py

```diff
@@ -164,8 +165,11 @@ def model_fn(features, labels, mode, params):  # pylint: disable=unused-argument
     initialized_variable_names = {}
     scaffold_fn = None
     if init_checkpoint:
-      (assignment_map, initialized_variable_names
-      ) = modeling.get_assignment_map_from_checkpoint(tvars, init_checkpoint)
+      # (assignment_map, initialized_variable_names
+      # ) = modeling.get_assignment_map_from_checkpoint(tvars, init_checkpoint)
+      assignment_map = {
+          "bert/": "bert/",
```

Comment:
@sgpyc - Is this the change that affects whether the optimized states are loaded from checkpoint?

Reply:
@sgpyc Please confirm the status of the optimizer states from checkpoint.

The reference has not used the optimizer state from the phase1 checkpoint in v0.7 and v1.0, so we should not be changing this now. In this case, true scale-vs-GA (the main purpose of this PR) should not be impacted by using default/zeroed velocities and momentums.

Reply:
I'm not pushing to merge this change, and just to show this as a potential place to look at.
Pervious and current data show whether to load the optimizer state has pretty minor impact on convergence, if there is any at all.

I would suggest to still allow both handlings (as in v0.7 & 1.0), although the reference doesn't load the optimizer states.